src/vs/workbench/services/workingCopy/test/browser/untitledFileWorkingCopy.test.ts

```diff
@@ -102,6 +102,7 @@ suite('UntitledFileWorkingCopy', () => {
 			uri,
 			basename(uri),
 			hasAssociatedFilePath,
+			false,
```

Comment:
We should also have a test here for scratchpads to ensure the manager passes the flag on accordingly.